.github/workflows/examples-efr32.yaml

```diff
@@ -106,6 +106,7 @@ jobs:
                 --enable-flashbundle \
                 --target efr32-brd2605a-light-skip-rps-generation \
                 --target efr32-brd4338a-lock-skip-rps-generation \
+                --target efr32-brd4343a-lock-skip-rps-generation \
```

Comment:
It isn't worth building the lock app again for a different board when it is the same board family and essentially builds the same files.
Ci is long enough already.

Reply:
removed for brd4343a